api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -18,31 +22,42 @@
     private final EligibilityClient client;
     private final EligibilityStatusCalculator eligibilityStatusCalculator;
 
+    private final Map<EligibilityStatus, ClaimStatus> eligibilityStatusMapper = ImmutableMap.of(
+            EligibilityStatus.ELIGIBLE, ClaimStatus.NEW,
```

Comment:
We're missing EligibilityStatus.INELIGIBLE -> REJECTED

Reply:
Added missing case.